.github/ISSUE_TEMPLATE/question.md

```diff
@@ -1,21 +1,12 @@
 ---
 name: Question
-about: Ask the Redis developers
+about: Ask the developers
 title: '[QUESTION]'
 labels: ''
 assignees: ''
 
 ---
 
-Please keep in mind that this issue tracker should be used for reporting bugs or proposing improvements to the Redis server.
+It is possible that your question was already asked here, or elsewhere, so please do a quick search before submitting. We do our best to answer questions but please note that this is not a free support service. It is a collaboration project. We help each other.
```

Comment:
I guess we'll add some more links when we have the name sorted, but this seems OK for the time being.